client/src/components/AdminContainer/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Layout, Button, Menu } from 'antd';
+import logo from '../../assets/images/logo.png';
+import './style.css';
+
+const { Header } = Layout;
+
+const AdminContainer = ({ buttonContent, contentComponent }) => {
+  return (
+    <>
+      <Layout>
+        <Header>
+          <div className="header-image">
+            <img src={logo} alt="logo" />
+          </div>
+          <div className="header-btn">
+            {buttonContent !== undefined && (
+              <Button type="primary" danger>
+                {buttonContent}
+              </Button>
+            )}
+          </div>
+        </Header>
+        <div className="container">
+          <div className="container-side">
+            <div className="side-content">
+              <Menu>
+                <Menu.Item key="1">
+                  <Link exact="true" to="/admin">
+                    Home
+                  </Link>
+                </Menu.Item>
+                <Menu.Item key="2">
+                  <Link exact="true" to="/admin/cohorts">
```

Comment:
@MohammedAlghazali I don't think Link components have an `exact` prop:
https://reacttraining.com/react-router/web/api/Link

Reply:
oh, yes, in fact, I want to use NavLink so I want benefit from its **activeClassName**,
then I decide to use the ordinary Link, and forget to remove it.